kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -201,20 +185,20 @@ deploy_kasfleetmanager() {
       "default": true
       "supported_instance_type":
         "standard": {}
-        "eval": {} 
+        "developer": {}
 ' \
+    -p STRIMZI_OPERATOR_SUBSCRIPTION_CONFIG="${STRIMZI_OPERATOR_SUBSCRIPTION_CONFIG}" \
+    -p KAS_FLEETSHARD_OPERATOR_SUBSCRIPTION_CONFIG="${KAS_FLEETSHARD_OPERATOR_SUBSCRIPTION_CONFIG}" \
     -p REPLICAS=1 \
-    -p KAFKA_CAPACITY_INGRESS_THROUGHPUT="${KAFKA_CAPACITY_INGRESS_THROUGHPUT}" \
-    -p KAFKA_CAPACITY_TOTAL_MAX_CONNECTIONS="${KAFKA_CAPACITY_TOTAL_MAX_CONNECTIONS}" \
-    -p KAFKA_CAPACITY_MAX_DATA_RETENTION_SIZE="${KAFKA_CAPACITY_MAX_DATA_RETENTION_SIZE}" \
-    -p KAFKA_CAPACITY_MAX_PARTITIONS="${KAFKA_CAPACITY_MAX_PARTITIONS}" \
-    -p KAFKA_CAPACITY_MAX_DATA_RETENTION_PERIOD="${KAFKA_CAPACITY_MAX_DATA_RETENTION_PERIOD}" \
-    -p KAFKA_CAPACITY_MAX_CONNECTION_ATTEMPTS_PER_SEC="${KAFKA_CAPACITY_MAX_CONNECTION_ATTEMPTS_PER_SEC}" \
     -p DEX_URL="http://dex-dex.apps.${K8S_CLUSTER_DOMAIN}" \
     -p TOKEN_ISSUER_URL="$(${KUBECTL} get route -n mas-sso keycloak -o jsonpath='https://{.status.ingress[0].host}/auth/realms/rhoas')" \
     -p ENABLE_OCM_MOCK=true \
```

Comment:
If the ocm service token `OCM_SERVICE_TOKEN` is set, this can be set to `false`. 
I propose its value be initialed based on the `OCM_SERVICE_TOKEN` being set/not-set.

Reply:
Updated to set `ENABLE_OCM_MOCK` dynamically depending on cluster provider (ocm/standalone).